src/Tizen.NUI.Components/Controls/Navigation/Navigator.cs

```diff
@@ -385,6 +387,7 @@ public Page Pop()
             }
 
             var curTop = Peek();
+            RemovePageToList(curTop);
```

Comment:
It seems that this API hides content of current page.
If there are multiple pages, then popped page is not immediately hidden. Instead, it shows pop animation and then disappears.
Therefore, this API should be called in Finished callback and if (navigationPages.Count == 1).

Isn't it required to call this API in PopWithTransition() and Remove()?

Reply:
I added Remove~ method(Currently, Hide~) to PushWithTransition() for  the disappeared page and Remove().
In Pop() and other methods, Remove() method is already called, so no need to call HideContentOfPage() again.
Please have a look.